misc/launchd/install.sh

```diff
@@ -19,5 +23,10 @@ if [ $? ]; then
 fi
 
 echo Loading ipfs-daemon
-launchctl load "$dest_dir/$plist"
+if [[ `uname -r` == "15.2.0" ]]; then
```

Comment:
this will get out of date soon, maybe just check that the first version is `>= 15`?